e2e-tests/cypress/support/appeal-submission-appeal-site-ownership/answerDidNotToldOtherOwnersAppeal.js

```diff
@@ -1,5 +1,5 @@
 module.exports = () => {
-  cy.get('#have-other-owners-been-told-no').click();
+  cy.get('#have-other-owners-been-told-2').click();
```

Comment:
Are we teenagers doing text - surely that should be too?

Reply:
Hah, I'm not actually sure this was meant to go in, David and I were figuring where the validation was going wrong.

Reply:
I'm not sure it was a necessary comment in this way. As has been mentioned elsewhere if we were to use the nunjucks component, the ID would be set to the plain id, then suffix 2, 3, etc. This is only the ID. The name and value are unaffected. It is only evident here because we are targeting ID and not a data-cy value or the name/value